index/secondary_index_manager.hh

```diff
@@ -99,6 +99,7 @@ public:
     bool is_index(view_ptr) const;
     bool is_index(const schema& s) const;
     bool is_global_index(const schema& s) const;
+    std::optional<sstring> vector_index_class(const schema& s) const;
```

Comment:
Let's return general custom index class and not distinguish vector index from other custom indexes (even though they don't exist) at this level.